### ISSUE TITLE
Missing output boxes.

### ISSUE THREAD

**Issue by GioLomia** [thtrieu/darkflow#1044]
Trained on my own data set. 8 classes, at step 11790 - loss 0.15480981767177582 and going down, I changed the number of classes in the config as well as the filters on the last layer to 65 (5*(classes+5)). Here is the command I used for training :
 !python3 flow --model cfg/yolov2-voc-8c.cfg --train --dataset ../Training_Set/Images --annotation ../Training_Set/Annotations --trainer adam --batch 10 --save 100 --keep 1 --gpu 1 --summary tensor --epoch 50 --load -1

However when I try to perform prediction with:
!python3 flow --imgdir ../Training_Set/Images --model cfg/yolov2-voc-8c.cfg --load -1 
The output is just the same images as the input ones. No boxes are generated even if I add 
--json at the end --json file comes up empty []. I am not sure what is happening.

**Comment by skofot** [comment]
Did you use pre-trained weights when starting your first training session? If not, download the corresponding weights [here](https://pjreddie.com/darknet/yolo/) and start a new training session where you use `--load filepath/file.weights` and then use `--load -1` only when continuing the training.

To see if you get any output at all, try lowering the threshold untill you get bounding boxes. This can be done with the command `--threshold X` where `0.0 < X < 1.0`. If this gives you bounding boxes that are somewhat correct your model could require more training or more data.

**Comment by GioLomia** [comment]
I am training on my custom data set, I am not sure why would it change anything, could you elaborate. Even when I lower the threshold down to 0.0 it still outputs nothing.

**Comment by mpky** [comment]
@GioLomia couple issues may be at play:

1. You need to make sure the cfg and the weights files actually match. I recommend using the weights found [here](https://drive.google.com/drive/folders/0B1tW_VtY7onidEwyQ2FtQVplWEU) rather than the ones on PJ Reddie's site.

2. How many images did you annotate and how similar are the objects? I imagine 8 classes takes quite a bit.

**Comment by GioLomia** [comment]
>how similar are the objects? I imagine 8 classes takes quite a bit.
I think that might be the problem because I am trying to detect animals in the wild that are camouflaged and the actual object density is low, meaning most of image is background, and the images are very similar because the images are cut up video feeds from the stationary wild life camera traps.

**Comment by GioLomia** [comment]
Ok it worked, when I lower my threshold to 0.005 it draws them. But it draws them everywhere. If I raise it any more they disappear.

**Comment by mpky** [comment]
@GioLomia Good it's detecting something at least. Might just need more training data

**Comment by GioLomia** [comment]
Maybe

**Comment by Avinash987** [comment]
I face the same issue, I've lowered the threshold to 0.001 and loss is 0.3.. still no bounding boxes.

**Comment by GioLomia** [comment]
I moved on to darknet so I am not using darkflow and it works great. From what I understand you have to first overfit on a small batch first and then train. Alternatively if you would prefer to train on darknet I have a label converter that transforms VOC labels into YOLO.

**Comment by xinyee1997** [comment]
@Avinash987 Do u solve the problem?

**Comment by GioLomia** [comment]
Yes

**Comment by ozanpkr** [comment]
> Yes
@GioLomia 
Hi brother.I have same ıssue.I use voc2012 train set for train yolov2.I have two classes ;person and car.My loss value nearly 0.00007 but I cannot see any bounding boxes when I did test.How can ı solve this problem?

**Comment by GioLomia** [comment]
@ghost-ronin I am not sure how to solve it on darkflow. I changed to using darknet instead.

**Comment by ozanpkr** [comment]
> @ghost-ronin I am not sure how to solve it on darkflow. I changed to using darknet instead.

Thanx for reply quickly.Last question ;) Can I use darknet weights for darkflow? or how can I use darknet on my windows?Dou you know any paper or blog about that?

**Comment by GioLomia** [comment]
@ghost-ronin I just virtualized Linux on my machine with cywig, it work really nice. You just install Cywig and put all your staff in the folder and just work on it as if its Linux. Darknet won't work on Windows, I think the weights won't work either.
